DESCRIPTION

```diff
@@ -52,6 +52,7 @@ Imports:
     glue,
     janitor,
     lifecycle,
+    magrittr,
```

Comment:
I would like to get rid of the magrittr pipe and never imported it because the pipe is reexported in dplyr and we need to watch the number of dependencies